platform/security/security-migratable/src/test/java/org/codice/ddf/security/migratable/impl/SecurityMigratableTest.java

```diff
@@ -47,68 +45,77 @@
 
     private static final String CRL_PROP_KEY = "org.apache.ws.security.crypto.merlin.x509crl.file";
 
-    private static final Path EXPORTED_REL_PATH = Paths.get("etc", "exported");
-
     private static final String DESCRIPTION = "Exports Security system files";
 
-    private static final Path FILE_CONTAINING_CRL_LOCATION = Paths.get("etc",
-            "ws-security",
-            "server",
-            "encryption.properties");
+    private static final Path SERVER_ENCRYPTION_PROPERTIES_PATH = Paths.get("etc", "ws-security",
+            "server", "encryption.properties");
 
-    private static final boolean IS_OPTIONAL = false;
+    private static final Path SERVER_SIGNATURE_PROPERTIES_PATH = Paths.get("etc", "ws-security",
+            "server", "signature.properties");
 
-    private Path ddfHome;
+    private static final Path ISSUER_ENCRYPTION_PROPERTIES_PATH = Paths.get("etc", "ws-security",
+            "issuer", "encryption.properties");
 
-    private Path exportDirectory;
+    private static final Path ISSUER_SIGNATURE_PROPERTIES_PATH = Paths.get("etc", "ws-security",
+            "issuer", "signature.properties");
 
-    @Before
-    public void setup() throws Exception {
-        ddfHome = Paths.get(DDF_BASE_DIR);
-        exportDirectory = ddfHome.resolve(EXPORTED_REL_PATH);
-    }
+    private static final Path EXPECTED_SERVER_SIGNATURE_CRL_PATH = Paths.get("ddf", "crl",
+            "serverSignature", "crl.pem");
+
+    private static final Path EXPECTED_SERVER_ENCRYPTION_CRL_PATH = Paths.get("ddf", "crl",
+            "serverEncryption", "crl.pem");
+
+    private static final Path EXPECTED_ISSUER_SIGNATURE_CRL_PATH = Paths.get("ddf", "crl",
+            "issuerSignature", "crl.pem");
+
+    private static final Path EXPECTED_ISSUER_ENCRYPTION_CRL_PATH = Paths.get("ddf", "crl",
+            "issuerEncryption", "crl.pem");
+
+    private static final boolean IS_OPTIONAL = false;
+
+    private static final Path DDF_HOME = Paths.get(DDF_BASE_DIR);
+
+    private static final Path EXPORT_DIRECTORY = DDF_HOME.resolve(Paths.get("etc", "exported"));
 
     @Test
     public void testExportValidRelativePaths() throws Exception {
```

Comment:
Should we have tests for the scenarios where `getJavaPropertyValue()` returns an empty value or throws a `MigrationException`?